client/src/components/subscription/subscriptionsList.js

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
 import SubscriptionListCard from './subscriptionListCard';
 
-const SubscriptionsList = ({ subscriptions, setActiveSubscription, handleDashboard, handleDelete, openDeleteModal }) => {
+const SubscriptionsList = ({ subscriptions, setActiveSubscription, handleDashboard, handleDelete, openDeleteModal, activeSubscription }) => {
   const subscriptionsList = subscriptions.map((subscription) => {
     const { subscriptionUuid } = subscription;
+    let className="list-group-item p-0 mt-2 list-group-item-action border border-dark rounded-1";
+    if (activeSubscription.subscriptionUuid === subscriptionUuid) {
+      className += " bg-primary";
+    }
```

Comment:
i've used [this library](https://www.npmjs.com/package/classnames) to better handle conditional classes but it's certainly not necessary. just something i want you to be aware of in case you wanted to look into it

Reply:
Thanks for bringing my attention to it, I'm good on this though.